{
    "created": 1644311730.3276992,
    "duration": 57.58277487754822,
    "exitcode": 1,
    "root": "C:\\AAIC\\pranav_automation",
    "environment": {
        "Python": "3.10.2",
        "Platform": "Windows-10-10.0.19042-SP0",
        "Packages": {
            "pytest": "6.2.5",
            "py": "1.11.0",
            "pluggy": "1.0.0"
        },
        "Plugins": {
            "html": "3.1.1",
            "json-report": "1.4.1",
            "metadata": "1.11.0"
        }
    },
    "summary": {
        "failed": 1,
        "total": 1,
        "collected": 1
    },
    "collectors": [
        {
            "nodeid": "",
            "outcome": "passed",
            "result": [
                {
                    "nodeid": "test_a.py",
                    "type": "Module"
                }
            ]
        },
        {
            "nodeid": "test_a.py",
            "outcome": "passed",
            "result": [
                {
                    "nodeid": "test_a.py::test_Openurl",
                    "type": "Function",
                    "lineno": 2
                }
            ]
        }
    ],
    "tests": [
        {
            "nodeid": "test_a.py::test_Openurl",
            "lineno": 2,
            "outcome": "failed",
            "keywords": [
                "test_a.py",
                "test_Openurl",
                "pranav_automation"
            ],
            "setup": {
                "duration": 1.3676838999381289,
                "outcome": "passed"
            },
            "call": {
                "duration": 55.943546700058505,
                "outcome": "failed",
                "crash": {
                    "path": "C:\\AAIC\\pranav_automation\\test_a.py",
                    "lineno": 16,
                    "message": "AttributeError: 'TypeError' object has no attribute 'message'"
                },
                "traceback": [
                    {
                        "path": "test_a.py",
                        "lineno": 16,
                        "message": "AttributeError"
                    }
                ],
                "longrepr": "setup = {'driver': <selenium.webdriver.chrome.webdriver.WebDriver (session=\"b49a3c970bea96f732974756bb229ed2\")>, 'url': 'https://appliedaiconsulting.com/'}\n\n    def test_Openurl(setup):\n        driver = setup[\"driver\"]\n        url = setup[\"url\"]\n        # import pdb\n        # pdb.set_trace()\n        try:\n            before_time = datetime.now().strftime('%H%M%S%f')  #  Timestamp\n            print(before_time)\n            driver.get(url)\n            now_time = datetime.now().strftime('%H%M%S%f')  #  Timestamp\n            print(now_time)\n>           time_diff = now_time - before_time\nE           TypeError: unsupported operand type(s) for -: 'str' and 'str'\n\ntest_a.py:14: TypeError\n\nDuring handling of the above exception, another exception occurred:\n\nsetup = {'driver': <selenium.webdriver.chrome.webdriver.WebDriver (session=\"b49a3c970bea96f732974756bb229ed2\")>, 'url': 'https://appliedaiconsulting.com/'}\n\n    def test_Openurl(setup):\n        driver = setup[\"driver\"]\n        url = setup[\"url\"]\n        # import pdb\n        # pdb.set_trace()\n        try:\n            before_time = datetime.now().strftime('%H%M%S%f')  #  Timestamp\n            print(before_time)\n            driver.get(url)\n            now_time = datetime.now().strftime('%H%M%S%f')  #  Timestamp\n            print(now_time)\n            time_diff = now_time - before_time\n        except Exception as e:\n>           print(e.message)\nE           AttributeError: 'TypeError' object has no attribute 'message'\n\ntest_a.py:16: AttributeError"
            },
            "teardown": {
                "duration": 0.005414300016127527,
                "outcome": "passed"
            }
        }
    ]
}